lib/utils/strings.dart

```diff
@@ -27,5 +30,11 @@ class Strings {
   static String myPlotDetailProfitabilityTitle = "Profitability";
   static String myPlotDetailCompanionPlantsTitle = "Companion Plants";
   static String myPlotDetailNonCompanionPlantsTitle = "Non-Companion Plants";
+    
+  static String documentFieldContent = "content";
```

Comment:
These are not "UI Strings" that are going to be localized neither shown to the user.
They are constants for firebase, check firebase_const file :)